commit 3ac667d5bc5eaf9accdf66f89915f99f487671bb (HEAD -> bugfix_util_format_date_missing_braces, personal-github/bugfix_util_format_date_missing_braces)
Author: Luke Dashjr <luke-jr+git@utopios.org>
Date:   Wed Feb 23 00:47:36 2022 +0000

    util: Add missing braces so format_date actually works with positive deltas

diff --git a/util.c b/util.c
index ce3ea0a..28d33ca 100644
--- a/util.c
+++ b/util.c
@@ -469,10 +469,11 @@ int format_date(time_t base, long delta, char* date, size_t date_len) {
         tt = base + delta;
         if (tt < base)
             tt = base;
-    } else
+    } else {
         tt = base - (-delta);
         if (tt > base)
             tt = base;
+    }
 
     if (gmtime_r(&tt, &tm) == NULL)
         return -1;
